crates/deterministic-wasi-ctx/Cargo.toml

```diff
@@ -12,11 +12,11 @@ categories = ["wasm"]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
 cap-primitives = "0.26.1"
-wasi-common = ">= 2.0.0, < 2.1.0"
```

Comment:
Couldn't we still allow wasi-common at version 2.0.0 to version 2.1.0 as well as between 3.0.0 and 3.1.0 with just this change? That is, do we need to bump the min version?